pydocstringformatter/run.py

```diff
@@ -47,9 +52,11 @@ def _format_file(self, filename: Path) -> bool:
         for index, tokeninfo in enumerate(tokens):
             new_tokeninfo = tokeninfo
 
+            formatter_options = vars(self.config)
             if utils._is_docstring(new_tokeninfo, tokens[index - 1]):
-                for formatter in formatting.FORMATTERS:
-                    new_tokeninfo = formatter.treat_token(new_tokeninfo)
+                for formatter in self.formatters:
+                    if formatter_options[formatter.name]:
```

Comment:
Can't we do `self.config.formatter`? That would save the assignment. I believe `Namespace` is special-cased by `mypy` not to generate any issues about this.

Reply:
I think if we do that we must control the formatters through the option parsing only and it would be harder to use the ``Run`` interface.

Reply:
Really? We would still call `utils._register_arguments_formatters` so any non-standard formatters get registered to the `ArgumentParser` (if we every extend the `Run` API to allow that, like I said probably not in this PR).
I just referred to this line. Instead of using `vars` to store the `Namespace` as a dict can't we just use `formatter.name` as the key on the `Namespace` object (stored on `self.config` directly)?

Reply:
Ho sorry, I thought that we were talking about ``self.formatter`` vs ``self.config.formatter``. Too much non ascii checker review this week !

Reply:
```
>                   if self.config["formatter"][formatter.name]:
E                   TypeError: 'Namespace' object is not subscriptable
```
```
>                   if self.config.formatter[formatter.name]:
E                   AttributeError: 'Namespace' object has no attribute 'formatter'
```

Calling vars multiple time seem like a performance issue ?

Reply:
Nvm we can do ``if getattr(self.config, formatter.name):``

Reply:
Hmm, what about:
```python
self.formatters = {i.name: i for i in formatting.FORMATTERS}

# and then

for formatter_name in self.formatter:
    if self.config.formatter_name:
        # do something
```

I'm not totally sure what type of keys `Namespace` allows, but it looked weird to call `vars` on an object that is basically a `dict` itself already.

Reply:
We need to use ``getattr(self.config, formatter.name):`` if we don't want to use ``vars``

Reply:
Fine with me!